test/tools/fuzzTesting/fuzzHelper.h

```diff
@@ -46,18 +46,27 @@ typedef boost::random::variate_generator<boost::mt19937&, boostUint64 > boostUIn
 struct RandomCodeOptions
 {
 public:
+	enum AddressType{
+		CALLONLY,
+		ACCOUNT,
+		ALL
```

Comment:
The coding rules say these should be
```
CallOnly,
Account,
All
```
but as I said I think
```
CallOnly,
InState,
All
```
would be better.